rskj-core/src/main/java/org/ethereum/config/SystemProperties.java

```diff
@@ -88,6 +88,8 @@
 
     public static final String PROPERTY_PRINT_SYSTEM_INFO = "system.printInfo";
 
+    public static final String PROPERTY_PREFLIGHT_CHECKS_SKIP_JAVA_VERSION_CHECK = "preflight.checks.skip.java.version.check";
```

Comment:
can we shorten the prop name? like "system.checkJavaVersion"?

Reply:
Yes, I think we can.

Reply:
Done!